src/locale/properties/register/register.xx.properties

```diff
@@ -74,3 +74,6 @@ register.moreInfo=X
 register.securityAndNotifications=X
 register.visibilityAndTerm=X
 register.visibilityRequired=X
+register.VisibilityPrivate2=X
```

Comment:
The properties `register.VisibilityPrivate1`, `register.VisibilityTrusted1` and `register.VisibilityEveryone1` are missing here and also in  the `rl` and `lr` properties files